src/connectors/index.ts

```diff
@@ -19,6 +22,14 @@ if (typeof INFURA_KEY === 'undefined') {
   throw new Error(`REACT_APP_INFURA_KEY must be a defined environment variable`)
 }
 
+const isInsideLedgerDappBrowser = () => {
+  const queryParams = parsedQueryString(window.location.search)
+  return IS_IN_IFRAME && queryParams.embed === 'ledgerdappbrowser'
+}
+if (isInsideLedgerDappBrowser()) {
+  window.ethereum = new IFrameEthereumProvider()
```

Comment:
would probably lazy load `IFrameEthereumProvider` here. Something like

```
if (isInsideLedgerDappBrowser()) {
  import('@ledgerhq/iframe-provider').then(({ IFrameEthereumProvider }) => {
    window.ethereum = new IFrameEthereumProvider()
  }
}
```

Reply:
Good point, updated the PR.